unit-tests/live/hdr/hdr_helper.py

```diff
@@ -64,12 +64,6 @@ def test_json_load(config, test_title):
             for control_key in key["controls"].keys():
                 key["controls"][control_key] = to_signed(key["controls"][control_key])
         test.check(hdr_config_read.get("hdr-preset") == config.get("hdr-preset"))
-
-        # check that the other keys are unchanged
-        for key in hdr_config_read.keys():
-            if key != "hdr-preset":
```

Comment:
Why remove all the checks? List params that change is expected and skip them, test others.
See example https://github.com/IntelRealSense/librealsense/blob/development/unit-tests/live/extrinsics/test-imu.py#L46

Reply:
As discussed, no need to check the values.